glide.yaml

```diff
@@ -13,3 +13,19 @@ import:
   version: kubernetes-1.10.3
 - package: k8s.io/gengo
 - package: github.com/xi2/xz
+- package: kubevirt.io/qe-tools
+  version: 13fc8573324cd2af42c88043b5a7c6e03bf39f95
+  repo: https://github.com/kubevirt/qe-tools.git
+- package: github.com/spf13/pflag
+- package: github.com/golang/protobuf
+  subpackages:
+  - proto
+- package: kubevirt.io/kubevirt
```

Comment:
we shouldn't be pulling in kubevirt ever into CDI. If there's something in kubevirt that CDI needs, we need to figure out a way to share that between both kubevirt and cdi, not have cdi depend on kubevirt.

This is going to cause problems because KubeVirt will actually depend on CDI soon. We don't want a circular dependency here.

Reply:
Yes, I realized I was a little too eager with my copy and paste. I will remove the references to kubevirt itself

Reply:
i figured as much, I just had to make a bunch of noise just in case :)

Reply:
Its my first patch with any actual GO in it. thanks for making noise and letting me learn.